lte/cloud/go/services/subscriberdb/load.go

```diff
@@ -231,3 +231,37 @@ func ConvertSubEntsToProtos(ent configurator.NetworkEntity, apnConfigs map[strin
 
 	return subData, nil
 }
+
+func LoadSuciProtosPage(
+	ctx context.Context, networkID string) ([]*lte_protos.SuciProfile, error) {
```

Comment:
Nit: no need to line break -- only line break at 120 (or even 130) characters

Reply:
done